low-level/framework/utils/utility.py

```diff
@@ -166,6 +167,48 @@ def get_current_time():
         """Returns the time as integer number in seconds since the epoch in UTC."""
         return int(time.time())
 
+    @staticmethod
+    def validate_storage_type_support(log, Error, storage_type):
```

Comment:
Please avoid adding global functions or functions in utils class. 
Think of classes and include this in the class. 
Refer to utils validator for examples.